### ISSUE TITLE
Exporting DrawRDKitBit structures

### ISSUE THREAD

**Issue by andt88** [rdkit/rdkit#2493]
Hi,
I am rather new to RDKit in python, and I have a couple of problems with the Draw module.
So my first one is, if there's any more elegant way than mine to save the drawn structures.

And the second one is how to find the available **kwargs for the DrawRDKitBit, since the problem I am having is that the circles drawn around aromatic atoms are somewhat cut off and don't always fit the image file (see below).

I'd really appreciate your help! 

![_1477_important_mol9_bit__weight_0 15](https://user-images.githubusercontent.com/36545428/59508837-94e57f00-8eaf-11e9-8ef9-29cbe7af6eae.png)

```
Chem.RDKFingerprint(mol, fpSize=self.fp_size, bitInfo=bi)
important=Draw.DrawRDKitBit(mol, cnt, bi)
filename = 'filename')
svg = open(filename + '.svg', 'w')
svg.write(important)
svg.close()
```

**Comment by greglandrum** [comment]
I believe this has recently been fixed in #2486
It will be in the 2019.03.3 release (hopefully happening next week)

**Comment by bp-kelley** [comment]
@andt88 if this doesn't answer your question, please reopen or open a new issue.
